Cargo.toml

```diff
@@ -50,6 +50,9 @@ bimap = "0.6"
 byteorder = "1"
 bytes = "1.1"
 chrono = "0.4"
+clickhouse-rs = { git = "https://github.com/suharev7/clickhouse-rs", rev = "c61904a", default-features = false, features = [
```

Comment:
Is it ok to use the most recent commit (as of today) from `clickhouse-rs` or should I use the most recent published version with a set of patches?

Reply:
Lets track the github repo here until we have a release, maybe add a comment that a release is pending.